ui-tests/src/test/resources/features/integrations/mongodb.feature

```diff
@@ -0,0 +1,386 @@
+@ui
+@mongodb
+@database
+@integrations-mongodb
+Feature: MongoDB
+
+  Background: Clean application state
+    Given clean application state
+    And deploy MongoDB 3.6 database
+    And connect to MongoDB "mongodb36"
+    And log into the Syndesis
+    And created connections
+      | MongoDB | mongodb36 | Mongo36 | Mongo description |
+
+
+  @mongodb-insert
+  @mongodb-consume
+  Scenario: MongoDB insert
+    Given create mongodb capped collection "capped_for_insert" with size 1000 and max 3
+    And create mongodb collection "insert_collection"
+    When navigate to the "Home" page
+    And click on the "Create Integration" link to create a new integration.
+    Then check visibility of visual integration editor
+    And check that position of connection to fill is "Start"
+
+    When select the "Mongo36" connection
+    And select "Mongo consumer" integration action
+    And fill in values by element data-testid
+      | collection               | capped_for_insert |
+      | tailtrackincreasingfield | _id               |
+    And click on the "Next" button
+
+    Then check visibility of page "Choose a Finish Connection"
+
+    When select the "Log" connection
+    And fill in values by element data-testid
+      | bodyloggingenabled | true |
+    And click on the "Next" button
+
+    And add integration step on position "0"
+    And select the "Mongo36" connection
+    And select "Mongo producer" integration action
+    And fill in values by element data-testid
+      | collection | insert_collection |
+      | operation  | Insert            |
+    And click on the "Next" button
+
+    And publish integration
+    And set integration name "Mongo to Mongo insert"
+    And publish integration
+
+    Then navigate to the "Integrations" page
+    And wait until integration "Mongo to Mongo insert" gets into "Running" state
+
+    When insert the following documents into mongodb collection "capped_for_insert"
+      | _id | value |
+      | id1 | v1    |
+    And sleep for 5 seconds
```

Comment:
woudn't it make more sense to have some wait in `verify that mongodb collection ...` rather than a random value here?

Reply:
Hmm, the problem here is that we put values into one mongo collection, the integration takes that and puts it into the other collection and then we verify. The delay is not in the verification, but in the time it takes the mongo connector to realize there's new data in the tracked collection.

Adding a wait in either the insert above or the verify below would mean we would have to way every time, even though it's only neccessary in this case.

Maybe I can add a comment to that effect here?

Reply:
So the transfer from one collection to other should be almost instant, because the connector is subscribed to the changes, it's just that this code is even faster. 

I'll add a wait, but I'll have to spend some more time on this, since using the `waitFor` would lose me the nice assertion message I have when the doc is not found

Reply:
yeah, the keywords `should be almost instant` are not 100% true on jenkins :grin:

Reply:
The `almost instant` part happens between two pods in the same namespace in openshift, so in this instance it's the code running on jenkins which _could_ be too fast. At least it was _sometimes_ when running locally, I guess it would work ok on jenkins.

Reply:
> I guess it would work ok on jenkins

so lets leave it this way and we will see